scripts/trim-low-abund.py

```diff
@@ -130,36 +137,77 @@ def main():
 
```

Comment:
`pass2filename = os.path.join(tempdir, pass2filename)` won't work if the user passed in absolute paths.